### PR TITLE
Improvement/add unit test

### PR DESCRIPTION
**Pull request by attila-at-hyper**

### CODE REVIEW

**Comment by 3lvis** [comment]
@attila-at-hyper this PR doesn't have commits :scream_cat:

**Comment by attila-at-hyper** [comment]
@3lvis: my bad, sorry. Now it does. Switching computers halfway doesn't always help... :)

**Comment by 3lvis** [comment]
@attila-at-hyper Would you mind fixing the merge conflict?

File: Pod/Tests/Tests.m [hunk 1]
```diff
@@ -7,9 +8,17 @@ @interface PodTests : XCTestCase
...
+    CardStackController *cardStackController = [CardStackController new];
+    
+    UIViewController *viewController1 = [UIViewController new];
+    UIViewController *viewController2 = [UIViewController new];
+    UIViewController *viewController3 = [UIViewController new];
+    cardStackController.viewControllers = @[viewController1, viewController2];
+    
+    XCTAssertEqual([cardStackController indexForViewController:viewController1], 0);
+    XCTAssertEqual([cardStackController indexForViewController:viewController2], 1);
+    XCTAssertEqual([cardStackController indexForViewController:viewController3], NSNotFound);
```
Comment (3lvis) [inline Pod/Tests/Tests.m:21]: Fantastic!

**Comment by attila-at-hyper** [comment]
@3lvis: sure, fixing now

**Comment by attila-at-hyper** [comment]
@3lvis: done

**Pull request merged**